src/storage/page_buffer.c

```diff
@@ -8326,12 +8326,23 @@ pgbuf_request_data_page_from_page_server (const VPID * vpid, log_lsa target_repl
 
   size += cublog::lsa_utils::get_packed_size (pac, size);
   size += vpid_utils::get_packed_size (pac, size);
+  size += pac.get_packed_int_size (size);
   std::unique_ptr<char[]> buffer (new char[size]);
 
   pac.set_buffer (buffer.get (), size);
   vpid_utils::pack (pac, *vpid);
   cublog::lsa_utils::pack (pac, target_repl_lsa);
 
+  PAGE_FETCH_MODE fetch_mode;
+  if (log_Gl.rcv_phase >= LOG_RECOVERY_UNDO_PHASE || log_Gl.rcv_phase == LOG_RESTARTED)
+    {
+      fetch_mode = OLD_PAGE;
+    }
+    else
+    {
+      fetch_mode = RECOVERY_PAGE;
+    }
+  pac.pack_int(fetch_mode);
```

Comment:
```suggestion
  pac.pack_int (fetch_mode);
```
check spacing all over